### ISSUE TITLE
Missing key SEQ for <INS>

### ISSUE THREAD

**Issue by brentp** [Illumina/paragraph#65]
Hi, I am trying to use paragraph to genotype manta calls. I get this error:
```
  Exception: Missing key SEQ for <INS> at 1:145235303; 
```
and that variant is indeed:
```
1       145235303       .       T       <INS>   0       .       END=145235303;SVTYPE=INS;CIPOS=0,14;CIEND=0,14;HOMLEN=14;HOMSEQ=GGAT
ATCAGGTTTT;LEFT_SVINSSEQ=GGATATCAGGTTTTCCTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTAAAGTGTGTGGGTCTTT;RIGHT_SVINSSEQ=GACCTCGTGATCCGCCTGCCTCGG
CCTCCCAAAGTGCTGGGATTACAGGCGTGAGCCACCGCGCCCGGCC
```
so it has left and right, but not `SEQ`. Is there something I can do to have paragraph genotype these? For example setting `SEQ = LEFT_SEQ + N*100 + RIGHT_SEQ` ?

**Comment by brentp** [comment]
Bump. I'm wondering if setting `SEQ = LEFT_SVINSSEQ + N*100 + RIGHT_SVINSSEQ` 
will work with the machinery in `paragraph`. I can preprocess the VCF to do this but want some idea that it does not violate some internal assumptions.

Is there also some way to genotype BND elements?

thanks!

**Comment by atongsa** [comment]
what is a homeseq in your info

**Comment by atongsa** [comment]
vim paragraph/lib/python3/grm/vcfgraph/vcfgraph.py

` edit_this_block
if alt == "<INS>":
`

**Comment by traxexx** [comment]
When the full insertion sequence cannot be assembled, Manta will report the sequence near breakpoints instead. Paragraph only uses sequences around breakpoints for genotyping, so @brentp solution should work.

**Comment by aksenia** [comment]
Nice hack! 
` vcf.info["LEFT_SVINSSEQ"] + ('N' * 200) + vcf.info["RIGHT_SVINSSEQ"]`

will throw an error though, as `vcf.info["LEFT_SVINSSEQ"] ` is returned as a tuple **("ACC...T",)**, so `vcf.info["LEFT_SVINSSEQ"][0]` is what will be needed to get to the seq.

**Comment by brentp** [comment]
Indeed. I am accumulating all the changes here: https://github.com/Illumina/paragraph/compare/master...brentp:bp-dev
